### PR TITLE
Allow to define starting seeds in external config file

### LINKED ISSUE #1461: Allow to define starting seeds in external config file
Right now Golem has:
* set of hardcoded bootstrap nodes to which it tries to connect after the start and this should stay like that. 
* set of previously known good nodes to which it tries to connect on start and this should stay like that
* one single seed node, which can be specifi in `appconfig.ini` to which Golem can try to connect on start.

### PR DESCRIPTION
**Pull request by jiivan**
fixes: #1461

### CODE REVIEW

File: golem/network/p2p/p2pservice.py [hunk 1]
```diff
@@ -1037,6 +1037,19 @@ def __sync_seeds(self, known_hosts=None):
...
+        for config_seed in self.config_desc.seeds.split(None):
+            try:
+                ip_address, port = config_seed.split(':', 1)
+                port = int(port)
+                # Verify that ip_address is valid.
+                # Throws ipaddress.AddressValueError.
+                ipaddress.ip_address(ip_address)
+            except ValueError:
+                logger.info("Invalid seed from config: %r. Ignoring.",
+                    config_seed)
```
Comment (maaktweluit) [inline golem/network/p2p/p2pservice.py]: wrong continuation, should match with `(`

File: tests/golem/core/test_simpleconfig.py [hunk 2]
```diff
@@ -1,61 +0,0 @@
-from golem.appconfig import CommonConfig
```
Comment (maaktweluit) [inline tests/golem/core/test_simpleconfig.py:1]: Why is this file removed?

**Review by maaktweluit — CHANGES_REQUESTED** [review]
LGTM!

Small lint fix and one question

File: tests/golem/core/test_simpleconfig.py (hunk 2 above)
Comment (jiivan) [reply tests/golem/core/test_simpleconfig.py:1]: CommonConfig is unused.

**Review by maaktweluit — APPROVED** [review]
